### PR TITLE
support delete topics using the kafka client

### LINKED ISSUE #347: [FEATURE] support delete topics using the kafka client
**Describe the solution you'd like**
sometimes we need to delete topics using the kafka client, like

`./bin/kafka-topics.sh --bootstrap-server 127.0.0.1:9092  --delete --topic test`

**Additional context**
delete topic will get a exception now.
image

### PR DESCRIPTION
**Pull request by dockerzhang**
fixes #347

### CODE REVIEW

**Comment by BewareMyPower** [comment]
Add a test to verify deleting an empty topic throws an `UnknownTopicOrPartitionException`. Here's an example:

```diff
diff --git a/tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java b/tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java
index 21cf192..1b393ca 100644
--- a/tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java
+++ b/tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java
@@ -376,6 +376,23 @@ public class KafkaRequestHandlerTest extends KopProtocolHandlerTestBase {
         verifyTopicsDeletedByPulsarAdmin(topicToNumPartitions);
     }
 
+    @Test(timeOut = 10000)
+    public void testDeleteNotExistedTopics() throws Exception {
+        Properties props = new Properties();
+        props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:" + getKafkaBrokerPort());
+
+        @Cleanup
+        AdminClient kafkaAdmin = AdminClient.create(props);
+        Set<String> topics = new HashSet<>();
+        topics.add("testDeleteNotExistedTopics");
+        try {
+            deleteTopicsByKafkaAdmin(kafkaAdmin, topics);
+            fail();
+        } catch (ExecutionException e) {
+            assertTrue(e.getCause() instanceof UnknownTopicOrPartitionException);
+        }
+    }
+
     @Test(timeOut = 10000)
     public void testCreateInvalidTopics() {
         Properties props = new Properties();
```

**Review by BewareMyPower — APPROVED** [review]
